### ISSUE TITLE
Merge select MD entries with CW ones

### ISSUE THREAD

**Issue by aarppe** [UAlbertaALTLab/cree-intelligent-dictionary#345]
There are a few frequent and central MD dictionary entries that have not been matched with CW ones due to non-conventional orthography. While a more permanent solution is in the works, we would manually want to add the following:

```
MD: wapimew - He sees him. -> CW: wâpamêw
MD: mitsow - He eats. -> CW: mîcisow
```

**Comment by aarppe** [comment]
Depending on where this is easiest to implement, we can either change the original XML files, or the internal database.

**Comment by aarppe** [comment]
Implemented the above, not only for the mentioned verbs but related words as well, in:

`~/giella/art/dicts/crk/crkeng_cw_md_200314.xml`

... which will eventually end up on sapir as well. This will require a new version of `engcrk*.xml `as well.

**Comment by aarppe** [comment]
The (slighty) revised: ` crkeng_cw_md_200314.xml` is now on sapir in: `/data/texts/crk/dicts/itwewina/`

**Comment by aarppe** [comment]
I also created the revised corresponding: `engcrk_cw_md_200317.xml`, also now on `sapir` in the directory mentioned above.

There's a few single quotes in the revised files, which I hope do not matter in the XML. If they do, one can convert those to the `&apos;` if needed.

**Comment by aarppe** [comment]
@Madoshakalaka The revised XML files, for both the crk-to-eng and eng-to-crk directions, adding a few select frequent MD items are now ready on sapir (see names and directories above). The revision resulted in also multi-word items getting proper ranks in `engcrk.xml`, when those are available in CW.

Before replacing the current content (which already works), please really do double-check programmatically that there are no issues with the XML code.

We can deal with the _itwêwina_ update within this issue, or keep here the creation of the XML files, and create a new issue for their uploading and integration.

**Comment by aarppe** [comment]
This should be resolved through #369.

**Comment by aarppe** [comment]
The modifications to the XML source appear now to be in place.
